engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -102,9 +103,13 @@ class WorkflowManagerActor(config: Config)
 
   private val donePromise = Promise[Unit]()
 
+  val isRestartable = config.getConfig("system").getBooleanOr("workflow-restart", default = true)
+  val serverMode = CromwellServer.isServerMode
```

Comment:
great question, it definitely isn't. I'm not sure where to add the hook of running in server mode. And I chose this because there was a note about the CromwellServer object: 
// Note that as per the language specification, this is instantiated lazily and only used when necessary (i.e. server mode)